sample-wallet-config.nix

```diff
@@ -0,0 +1,24 @@
+# If any customization is required, copy this file to
```

Comment:
This file needs to be in `/scripts/launch/connect-to-cluster` for this code to work

Reply:
If we were using JSON that would be true, but the path resolution works about when `custom-wallet-config.nix` is in the root of the repo and paths are specified relative to that directory.

Reply:
Ah, you're right.